lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -657,17 +824,41 @@ describe('NxCombobox', function() {
         const noValidationErrorsMinimalProps:Props<string | number> = nonPristineMinimalProps,
             quickRender = rtlRender(NxCombobox, noValidationErrorsMinimalProps);
 
-        it('has empty validation alert and no a11y error message', function() {
+        it('has no validation alert and no a11y error message', function() {
           const component = quickRender();
 
-          expect(component.getAllByRole('alert')[0]).toBeEmptyDOMElement();
+          expect(component.queryByRole('alert')).not.toBeInTheDocument();
           expect(component.getByRole('combobox')).not.toHaveErrorMessage();
         });
 
         it('does not set aria-invalid on the combobox', function() {
           expect(quickRender().getByRole('combobox')).not.toHaveAttribute('aria-invalid', 'true');
         });
 
+        describe('when in a form with showValidationErrors', function() {
+          function quickRender(extraProps?: Partial<Props>) {
+            const renderResult = render(
+              <NxForm showValidationErrors onSubmit={() => {}}>
+                <NxCombobox data-testid="combobox" { ...validatableMinimalProps } { ...extraProps } />
```

Comment:
same as above: https://github.com/sonatype/sonatype-react-shared-components/pull/789/files#r976899751

Reply:
> does the data-testid still need to be here?

No. https://github.com/sonatype/sonatype-react-shared-components/commit/98bd2530253ee6eb992af4a172d2c524ed0e8c85

Generally it's not helpful to comment every instance separately of a similar problem.  A better pattern when you see something like this is to edit your comment on the first instance to indicate that you see a need for the same fix throughout the file.

Reply:
> > does the data-testid still need to be here?
> 
> No. [98bd253](https://github.com/sonatype/sonatype-react-shared-components/commit/98bd2530253ee6eb992af4a172d2c524ed0e8c85)
> 
> Generally it's not helpful to comment every instance separately of a similar problem. A better pattern when you see something like this is to edit your comment on the first instance to indicate that you see a need for the same fix throughout the file.

Sounds good, wasn't sure which was the better approach. I'll edit that next time.